### PR TITLE
feature 2

### PR DESCRIPTION
**Pull request by duc-van-cloud**

### CODE REVIEW

File: hw.cpp [hunk 1]
```diff
@@ -1,6 +1,11 @@
 #include <stdio.h>
 #include <conio.h>
 
+int feature_2()
+{
+	return 0;
```
Comment (duc-van-cloud) [inline hw.cpp]: @trungng-tibco please help to review content

File: hw.cpp [hunk 2]
```diff
@@ -1,6 +1,11 @@
 #include <stdio.h>
 #include <conio.h>
```
Comment (duc-van-cloud) [inline hw.cpp:3]: you should include ticket number @trungng-tibco

File: hw.cpp (hunk 1 above)
Comment (trungng-tibco) [inline hw.cpp]: I think this should return 1

**Review by trungng-tibco — CHANGES_REQUESTED** [review]
I think it should return 1

File: hw.cpp (hunk 1 above)
Comment (duc-van-cloud) [reply hw.cpp]: Returning 1

**Review by trungng-tibco — APPROVED** [review]

**Pull request merged**